bin/python-stripped-env

```diff
@@ -30,8 +30,28 @@
 # -u  Treat unset variables as an error when substituting.
 set -e -u
 
-# actual python binary that we'll call
-PYTHON=/usr/bin/python
+# default python interpreter
+DEFAULT_PYTHON=/usr/bin/python
+
+# check arguments for any known python interpreter
+first_arg=${1:-false}
+case "$first_arg" in
+    python*)
+        # use first argument if it is an available interpreter
+        if command -v $1 &> /dev/null; then
+            PYTHON=$(which $1)
+        else
+            # fallback to default option
+            PYTHON=$DEFAULT_PYTHON
```

Comment:
set this before the case, as a default, and get rid of the else and faalback case statement

Reply:
good point, fixed in https://github.com/hpcugent/vsc-install/pull/175/commits/5d04d84163238dab95f5916afb6800e20b448070